Source/Core/Math.js

```diff
@@ -583,6 +583,108 @@ define([
         return absDiff <= absoluteEpsilon || absDiff <= relativeEpsilon * Math.max(Math.abs(left), Math.abs(right));
     };
 
+    /**
+     * Determines if the left value is less than the right value. If the two values are within
+     * <code>absoluteEpsilon</code> of each other, they are considered equal and this function returns false.
+     *
+     * @param {Number} left The first number to compare.
+     * @param {Number} second The second number to compare.
+     * @param {Number} absoluteEpsilon The absolute epsilon to use in comparison.
+     * @returns {Boolean} <code>true</code> if <code>left</code> is less than <code>right</code> by more than
+     *          <code>absoluteEpsilon<code>. <code>false</code> if <code>left</code> is greater or if the two
+     *          values are nearly equal.
+     */
+    CesiumMath.leftIsLessThanRight = function(left, right, absoluteEpsilon) {
```

Comment:
Should these just be `lessThan`, `lessThanOrEqualTo`, etc?  We don't call the function `leftIsEqualToRight`

Reply:
I thought may names made the user clearer. Order doesn't matter in `equals`, so being precise is less important there. But if you all feel strongly about it I'm happy to rename them.

Reply:
I do like this because otherwise I'd have to go look up the documentation whenever using this to check which way it's comparing them. This way it's self-documenting.

Reply:
Right than left is standard, so this seems a little pedantic.  In `Check` we just call these functions `greaterThan` `lessThan` etc https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/Check.js#L89

Reply:
That's fair. Sounds like we should change it for consistency here then @kring .

Reply:
I honestly don't care either way, but `Check` isn't public so I wouldn't necessarily use that as the decision maker.

Reply:
I thought we had functions like this all over the code, but to might surprise, JulianDate is the only place I could find them (outside of Check).  While I'm on board with renaming them for consistency, I don't want it to hold up this PR which really needs time to stew in master.

If someone wants to open a separate PR to clean things up before the next release, I'll review and merge it ASAP.